python/ray/worker.py

```diff
@@ -2036,12 +1951,32 @@ def connect(node,
         # driver task.
         worker.task_context.current_task_id = driver_task_spec.task_id()
 
-    worker.raylet_client = ray._raylet.RayletClient(
+    redis_address_components = node.redis_address.split(":")
+    redis_address = redis_address_components[0]
+    if len(redis_address_components) > 1:
+        redis_port = int(redis_address_components[1])
+    else:
+        redis_port = 0
```

Comment:
When is this used? Please add a comment about it.

Reply:
Leaving validation to `node.py`